packages/sql-drizzle/src/internal/patch.ts

```diff
@@ -73,6 +73,8 @@ export const makeRemoteCallback = Effect.gen(function*() {
       : statement.withoutTransform
     if (method === "get") {
       effect = Effect.map(effect, (rows) => rows[0] ?? [])
+    } else if (method === "execute") {
+      effect = Effect.map(effect, (rows) => rows.length ? rows : [{}])
```

Comment:
I stuggled a bit with that part. It turns out that when `method === "execute"` `drizzle expects a non empty array with at least one element in it.
This seems to solve the test, but hard to say if it does works well in other cases.

References: 
- https://github.com/drizzle-team/drizzle-orm/tree/main/examples/mysql-proxy
- https://github.com/drizzle-team/drizzle-orm/blob/9e81def6dbe33e54ef96f5960cf87ec829b7414f/drizzle-orm/src/mysql-proxy/session.ts#L120-L123